skins/oasis/css/core/ads.scss

```diff
@@ -60,20 +62,54 @@ $uap-video-player-button-size: 24px;
 		overflow: hidden;
 	}
 
+	.provider-container {
+		position: relative;
+	}
+
 	&.slot-responsive {
 		width: 100%;
 
-		.provider-container {
-			position: relative;
+		.provider-container iframe {
+			height: 100%;
+			left: 0;
+			position: absolute;
+			width: 100%;
+			top: 0;
+		}
+	}
+
+	.vpaid-container {
+		left: 0;
+		position: absolute;
+		top: 0;
+		width: 100%;
+
+		video, & > div {
+			height: 100%;
+			left: 0;
+			position: absolute;
+			width: 100%;
+			top: 0;
```

Comment:
What do you think about moving
```
left: 0;
position: absolute;
top: 0;
width: 100%;
```
to a mixin and use it in those couple of places? It will be more readable and reusable